decidim-core/app/cells/decidim/amendable/announcement_cell.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Decidim::Amendable
+  # This cell renders the callout with information about the state of the emendation
+  class AnnouncementCell < Decidim::ViewModel
+
+    def show
+      cell "decidim/announcement", announcement
+    end
+
+    private
+
+    def announcement
+      {
+      announcement: emendation_message,
+      callout_class: state_classes
+      }
+    end
+
+    def emendation_message
+      case model.emendation_state
+      when "accepted"
+        t(:accepted,
+          scope: "decidim.amendments.emendation.announcement",
+          amendable_type: amendable_type,
+          amendable_link: amendable_link,
+          announcement_date: announcement_date
+        )
+      when "rejected"
+        t(:rejected,
+          scope: "decidim.amendments.emendation.announcement",
+          amendable_type: amendable_type,
+          amendable_link: amendable_link,
+          announcement_date: announcement_date,
+          publish_as_button: publish_as_button
+        )
+      when "evaluating"
+        t(:evaluating,
+          scope: "decidim.amendments.emendation.announcement",
+          amendable_type: amendable_type,
+          amendable_link: amendable_link
+        )
+      end
+    end
+
+    def amendable_link
+      link_to resource_locator(model.amendable).path do
+        %{<strong>#{model.amendable.title}</strong>}
+      end
+    end
+
+    def amendable_type
+      @label ||= t(model.class.model_name.i18n_key, scope: "activerecord.models", count: 1).downcase
+    end
+
+    def announcement_date
+      case model.emendation_state
+      when "accepted"
+        Time.now
```

Comment:
`Time.current`

Reply:
this was temporary but, thanks for the tip